src/rules/getter_return.rs

```diff
@@ -341,6 +341,8 @@ mod tests {
          { bar: { get: function() { return true; } } });"#,
       r#"Object.defineProperties(foo,
          { bar: { get: function () { ~function() { return true; }(); return true; } } });"#,
+      r#"Refrect.defineProperty(foo, "bar", { get: function () { return true; } });"#,
+      r#"Refrect.defineProperty(foo, "bar", { get: () => { return true; } });"#,
```

Comment:
```suggestion
      r#"Reflect.defineProperty(foo, "bar", { get: function () { return true; } });"#,
      r#"Reflect.defineProperty(foo, "bar", { get: () => { return true; } });"#,
```

typo

Reply:
Oh... Thank you for pointing it out 🙇‍♂️